build.zig

```diff
@@ -156,6 +166,19 @@ pub fn build(b: *zbs.Builder) !void {
     }
 }
 
+fn writePkgconfigFile(writer: anytype, prefix: []const u8, v: []const u8) !void {
```

Comment:
I don't think splitting this out into a function really makes sense, it's only used in one place.

Reply:
Yes you're right, it was because my first version when I was testing was different and need a function but now it doesn't makes sense